maths/number_of_digits.py

```diff
@@ -10,11 +10,18 @@ def num_digits(n: int) -> int:
     5
     >>> num_digits(123)
     3
+    >>> num_digits(0)
+    1
+    >>> num_digits(-1)
+    1
```

Comment:
Nice to have comprehensive tests!!  Please also add a larger negative number.

Reply:
I did this. Please review again :)